platform_unix.cpp

```diff
@@ -51,6 +51,11 @@ char *show_file_picker() {
 					"_Open", GTK_RESPONSE_ACCEPT,
 					NULL );
 
+	// Filter file types
+	GtkFileFilter *filter = gtk_file_filter_new();
+	gtk_file_filter_add_pattern(filter, "*.brd");
+	gtk_file_chooser_add_filter(dialog, filter);
```

Comment:
For feature parity with windows, you should also use `gtk_file_filter_set_name(filter, "BRD Files");` and consider adding a second GtkFileFilter with pattern `"*"` and name `"All Files"`